config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,8 @@
+# This file is overwritten on deploy
+
+Sidekiq.configure_client do |config|
+  config.redis = { :namespace => 'collections-publisher' }
```

Comment:
It would be good to update this to read the redis config from `config/redis.yml`.  See publisher for example.